helm.sh/_posts/2018-09-07-new-gov-and-elections.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "New Governance And Elections"
+permalink: "/blog/new-gov-and-elections/"
+author: "Matt Farina"
+authorlink: "https://mattfarina.com"
+---
+
+When Helm moved from being under Kubernetes to a CNCF project there was a need to create a governance structure. While under Kubernetes we relied on Kubernetes governance. To handle this we setup a [provincial governance](https://github.com/helm/community/blob/aa0586011786dfbc3993e7edd959a841241c96e3/governance/provisional-governance.md) with a goal of creating a long term one. After a few months we are happy to announce that the new governance structure as been written and approved.
```

Comment:
I'd bring a little more clarity to the first sentence by replacing it with something like this: 
`Being a top level incubating CNCF project requires having a governance structure to ensure that there is a publicly documented process for making decisions regarding the project and the community. While Helm was under Kubernetes, we relied on Kubernetes governance. As part of the transition to CNCF, the Helm project is required to have its own governance structure. To handle this we set up...`

Reply:
Good call. Updated.